README.md

```diff
@@ -1 +1,48 @@
-# Zomato-Clone
\ No newline at end of file
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/SuperAayush/Zomato-Clone)
+
+Gitpod can provide a simple way to clone a repository on the browser itself without fork. You can configufre the repository on the browser version of VS Code without worrying about the dependencies to install.
+
+<h1 align="center">Zomato-Clone</h1>
+
+<p aign="center">
+  <p align="center">A clone app of Zomato to practice HTML5 & CSS3.
+    </p>
+  <p align="center">Website: <a href="https://superaayush.github.io/Zomato-Clone/">Zomato-Clone</a></p>
+</p>
+
+It is a beginner friendly project to start work upon, where you can learn:
+- How to raise an issue?
+- How to fork and work upon a repository?
+- How to make a PR?
+
+<br>
+
+![Screenshot](/Images/ss1.png)
+
+<br>
+
+![Screenshot](/Images/ss4.png)
+
+<br>
+
+## Technical Details
+**Languages:**
+```HTML5, CSS3, JavaScript```
+
+**Platforms:** 
+```GitHub Pages```
+
+<hr>
+
+## Contributing
+
+Contributions are always welcome!
+
+See [CONTRIBUTIONS.md](https://github.com/SuperAayush/Zomato-Clone/blob/main/CONTRIBUTING.md) for ways to get started.
+
+Please adhere to this project's ```code of conduct```.
```

Comment:
Please add the link to the code of conduct file 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Ok looking for it....!

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Efficient